kythe/go/extractors/bazel/extractor.go

```diff
@@ -68,10 +68,11 @@ import (
 // Finally, the "Fixup" stage gives the caller a final opportunity to edit the
 // resulting compilation record before it is returned.
 type Config struct {
-	Corpus   string          // the default corpus label to use
-	Language string          // the language label to apply
-	Rules    vnameutil.Rules // rules for rewriting file VNames
-	Verbose  bool            // whether to emit verbose (per-file) logging
+	Corpus      string          // the default corpus label to use
+	Language    string          // the language label to apply
+	Rules       vnameutil.Rules // rules for rewriting file VNames
+	Verbose     bool            // whether to emit verbose (per-file) logging
+	ProtoFormat string          // output format of compilation unit (json or proto)
```

Comment:
Why use a string here and not the enum?

Reply:
Good point. Changed this to use `kzip.Encoding`

Reply:
I'd rename the field to something like "KzipEncoding" over "ProtoFormat".

Reply:
Renamed fields